plotly/offline/offline.py

```diff
@@ -577,31 +611,61 @@ def plot(figure_or_data, show_link=True, link_text='Export to plot.ly',
         figure_or_data, config, validate,
         '100%', '100%', global_requirejs=False)
 
+    # Build resize_script
     resize_script = ''
     if width == '100%' or height == '100%':
         resize_script = _build_resize_script(plotdivid)
 
+    # Process include_plotlyjs and build plotly_js_script
+    include_plotlyjs_orig = include_plotlyjs
     if isinstance(include_plotlyjs, six.string_types):
         include_plotlyjs = include_plotlyjs.lower()
 
     if include_plotlyjs == 'cdn':
-        plotly_js_script = """\
+        plotly_js_script = _window_plotly_config + """\
 <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>"""
     elif include_plotlyjs == 'directory':
-        plotly_js_script = '<script src="plotly.min.js"></script>'
+        plotly_js_script = (_window_plotly_config +
+                            '<script src="plotly.min.js"></script>')
     elif (isinstance(include_plotlyjs, six.string_types) and
           include_plotlyjs.endswith('.js')):
-        plotly_js_script = '<script src="{url}"></script>'.format(
-            url=include_plotlyjs)
+        plotly_js_script = (_window_plotly_config +
+                            '<script src="{url}"></script>'.format(
+                                url=include_plotlyjs_orig))
     elif include_plotlyjs:
         plotly_js_script = ''.join([
+            _window_plotly_config,
             '<script type="text/javascript">',
             get_plotlyjs(),
             '</script>',
         ])
     else:
         plotly_js_script = ''
 
+    # Process include_mathjax and build mathjax_script
+    include_mathjax_orig = include_mathjax
+    if isinstance(include_mathjax, six.string_types):
+        include_mathjax = include_mathjax.lower()
+
+    if include_mathjax == 'cdn':
+        mathjax_script = (
+            '<script src="{url}?config=TeX-AMS-MML_SVG"></script>'.format(
```

Comment:
Add `STIX_Web` font config from `mathjax_config`.  This way appearance of exported html will match `iplot` and `FigureWidget`

Reply:
Is there an example on how to use `include_mathjax='cdn'` in a Jupyter notebook? For instance where would I enable LaTeX in plotly in the following block?
```python
import plotly.offline as py
py.init_notebook_mode()
import plotly.graph_objs as go
from ipywidgets import interact
fig = go.FigureWidget()
```
I don't seem to get LaTeX working in the resulting interactive plots.
Thank you.

Reply:
Hi @dealmeidavf ,

`include_mathajx` is only needed for `plotly.offline.plot`.  For `FigureWidget` and `plotly.offline.iplot` Jupyter's internal MathJax should be used automatically.

```python
import plotly.graph_objs as go
go.FigureWidget(layout={'title': '$\LaTeX$'})
```
![newplot 16](https://user-images.githubusercontent.com/15064365/48671229-efbe2000-eaf2-11e8-9d1c-68977c30b563.png)

Could you open a new issue with your browser info, and whether you're using the classic notebook of JupyterLab? There are some issues with FireFox still, so try it out with Chrome if you haven't. Also, in JupyterLab you sometimes need to run a markdown cell containing LaTeX first so that the MathJax is initialized.

Reply:
Thank you. I am using Jupyter notebook in the anaconda navigator on a Mac.  LaTeX does not work on a Mac Firefox. It works on Safari. I am new to plotly but a user of matplotlib. The interactive plots in plotly using ipywidgets are amazing!